app/domain/authentication/validate_security.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Authentication
+  ValidateSecurity = CommandClass.new(
+    dependencies: {
+      security: ::Authentication::Security.new
+    },
+    inputs: %i(input enabled_authenticators)
+  ) do
+
```

Comment:
same thing here, just move line 18 directly inside call and delete the private method `validate_security`.